reference-architecture/azure-ansible/store.json

```diff
@@ -0,0 +1,266 @@
+              "lun" : 5,
+              "name" : "datadisk6",
+              "vhd" : {
+                "uri" : "[concat('http://', parameters('sa'), '.blob.core.windows.net/openshiftvmachines/', parameters('vmName'), '_datadisk6.vhd')]"
+              }
+            },
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "128",
+              "lun" : 6,
+              "name" : "datadisk7",
+              "vhd" : {
+                "uri" : "[concat('http://', parameters('sa'), '.blob.core.windows.net/openshiftvmachines/', parameters('vmName'), '_datadisk7.vhd')]"
+              }
+            },
+            {
+              "caching" : "None",
+              "createOption" : "Empty",
+              "diskSizeGB" : "128",
+              "lun" : 7,
+              "name" : "datadisk8",
+              "vhd" : {
+                "uri" : "[concat('http://', parameters('sa'), '.blob.core.windows.net/openshiftvmachines/', parameters('vmName'), '_datadisk8.vhd')]"
+              }
+            }
+          ]
+        },
+        "networkProfile" : {
+          "networkInterfaces" : [
+            {
+              "id" : "[resourceId('Microsoft.Network/networkInterfaces', concat(parameters('vmName'), 'nic'))]"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "type" : "Microsoft.Compute/virtualMachines/extensions",
+      "name" : "[concat(parameters('vmName'),'/installcustomscript')]",
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[resourceGroup().location]",
+      "tags" : {
+        "displayName" : "VirtualMachineCustomScriptExtension"
+      },
+      "dependsOn" : [
+        "[concat('Microsoft.Compute/virtualMachines/', parameters('vmName'))]"
+      ],
+      "properties" : {
+        "publisher" : "Microsoft.OSTCExtensions",
+        "type" : "CustomScriptForLinux",
+        "typeHandlerVersion" : "1.3",
+        "settings" : {
+          "fileUris" : [
+            "[concat(parameters('baseTemplateUrl'), 'store.sh')]"
+          ],
+          "commandToExecute" : "[ concat('sh store.sh ', resourceGroup().name, ' ', parameters('adminUsername'), ' ',   variables('sQuote'),  parameters('sshPrivateData'), variables('sQuote'),  ' ', variables('sQuote'), parameters('sshKeyData'), variables('sQuote')  ) ]"
+        }
+      }
+    },
+    {
+      "type" : "Microsoft.Network/networkSecurityGroups",
+      "name" : "[concat(parameters('vmName'), 'nsg')]",
+      "tags" : {
+        "displayName" : "NetworkSecurityGroup"
+      },
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[resourceGroup().location]",
+      "properties" : {
+        "securityRules" : [
+          {
+            "name" : "default-allow-ssh",
+            "properties" : {
+              "protocol" : "Tcp",
+              "sourcePortRange" : "*",
+              "destinationPortRange" : "22",
+              "sourceAddressPrefix" : "*",
```

Comment:
But does this mean that access to hosts can be done directly bypassing the bastion?